BaseLib/FileTools.cpp

```diff
@@ -141,11 +141,11 @@ std::string extractPath(std::string const& pathname)
         return "";
     return pathname.substr(0, pos + 1);
 }
-static const char * pathSeparator =
+static const char pathSeparator =
```

Comment:
A backslash as path separator would definitely be wrong for unix. Furthermore having a string of length > 1 would make some functions in this file slightly more complicated. So I don't want to change it, and the ifdef has been there also before this PR.